deltacat/compute/compactor_v2/compaction_session.py

```diff
@@ -694,3 +536,274 @@ def merge_input_provider(index, item):
         round_completion_file_s3_url,
         is_inplace_compacted,
     )
+
+
+def _process_merge_results(
```

Comment:
This method can be used from within the `_run_merge`? That way compaction session gets the processed results to work with.